pkg/systemd/logDetails.jsx

```diff
@@ -93,29 +110,26 @@
     }
 
     loadProblem(entry) {
-        const problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
-        const service = problems_client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
-        const problems = problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
-
-        problems.wait(() => {
-            service.GetProblems(0, {})
-                    .then(problem_paths => {
-                        const fields = [entry.PROBLEM_DIR, entry.PROBLEM_DUPHASH, entry.PROBLEM_UUID];
-                        let path = null;
-                        problem_paths.some(pth => {
-                            const p = problems[pth];
-                            if (p && (fields.indexOf(p.ID) > 0 || fields.indexOf(p.UUID) || fields.indexOf(p.Duphash))) {
-                                path = p;
-                                return true;
-                            } else {
-                                return false;
-                            }
-                        });
-
-                        this.setState({ entry: entry, loading: false, error: "", problemPath: path, abrtService: service });
-                    })
-                    .catch(err => this.setState({ entry: entry, loading: false, error: err.toString() }));
-        });
+        const client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
+        const service = client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
+
+        get_problems(service)
+                .then(problems => {
+                    const fields = [entry.PROBLEM_DIR, entry.PROBLEM_DUPHASH, entry.PROBLEM_UUID];
+                    let path = null;
+                    Object.keys(problems).some(pth => {
+                        const p = problems[pth];
+                        if (p && (fields.indexOf(p.ID) > 0 || fields.indexOf(p.UUID) || fields.indexOf(p.Duphash))) {
+                            path = p;
+                            return true;
+                        } else {
+                            return false;
+                        }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18052-20221216-143043-67e35acc-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)